comms/src/inbound_message_service/comms_msg_handlers.rs

```diff
@@ -79,19 +95,48 @@ impl<P: PublicKey> DispatchResolver<CommsDispatchType, MessageContext<P>> for In
     }
 }
 
-fn handler_handle<PubKey: PublicKey>(_message_context: MessageContext<PubKey>) -> Result<(), DispatchError> {
+fn handler_handle<PubKey, DispKey, DispRes>(
+    message_context: MessageContext<PubKey, DispKey, DispRes>,
+) -> Result<(), DispatchError>
+where
+    PubKey: PublicKey,
+    DispKey: DispatchableKey,
+    DispRes: DispatchResolver<DispKey, DomainMessageContext<PubKey>>,
+{
     // TODO: Pass message to DHT and/or Domain Dispatcher
 
-    Ok(())
+    // Change the MessageContext into the DomainMessageContext and submit to the domain dispatcher
+    let message = message_context.message_data.message_envelope.message_body().unwrap(); // TODO: handle MessageError correctly
```

Comment:
Think you missed this TODO

Reply:
Good catch. I mapped the error to a DispatchError.